Configuration/Generator/python/XiMinus_13p6TeV_SoftQCDInel_TuneCP5_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 from Configuration.Generator.Pythia8CommonSettings_cfi import *
-from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
 
 
 generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
Is this file the same as 
https://github.com/cms-sw/cmssw/pull/39243

If yes, I can close that PR. And please update to `Pythia8ConcurrentGeneratorFilter`.

Reply:
it contains explicitly CP5

Reply:
I think we keep calling CP5 in production also,
https://cms-pdmv.cern.ch/mcm/requests?member_of_campaign=Run3Summer22GS&page=0&shown=549756076159

Reply:
well, is that the right decision?

Reply:
@cms-sw/pdmv-l2 @cms-sw/generators-l2 may clarify for the dataset/fragment names.